src/action/autopilot.js

```diff
@@ -48,6 +55,69 @@ class AtplAction {
       this._notification.display({ msg: 'Error toggling autopilot', err });
     }
   }
+
+  /**
+   * Update autopilot's channel scores so it knows whom to open channels with.
+   * @return {Promise<undefined>}
+   */
+  async updateAutopilotScores() {
+    try {
+      const nodeScores = await this.fetchBosScores();
+      await this._grpc.sendAutopilotCommand('setScores', {
+        heuristic: 'externalscore',
+        scores: nodeScores,
+      });
+    } catch (err) {
+      log.error('Updating autopilot scores failed', err);
+    }
+  }
+
+  /**
+   * Fetch bos scores, persist them, and return them in a list.
+   * @return {Object}
+   */
+  async fetchBosScores() {
+    const networkStr =
+      (await this._getNetwork()) === 'testnet' ? 'testnet' : '';
+    try {
+      const uri = `https://nodes.lightning.computer/availability/btc${networkStr}.json`;
+      const response = checkHttpStatus(await fetch(uri));
+      const json = await response.json();
+      const scores = this._formatAtplScores(json.scores);
+      if (networkStr === '') this._store.settings.atplScores = scores;
+      else this._store.settings.atplTestnetScores = scores;
+      this._db.save();
+      return scores;
+    } catch (err) {
+      log.error('Failed to fetch bos scores', err);
+      let cachedScores =
+        networkStr === ''
+          ? this._store.settings.atplScores
+          : this._store.settings.atplTestnetScores;
+      if (cachedScores !== {}) {
+        return cachedScores;
+      } else {
+        throw new Error(
+          'Failed to fetch bos scores and no scores are cached',
+          err
+        );
+      }
+    }
+  }
+
+  _formatAtplScores(jsonScores) {
+    let scores = {};
+    for (let node of jsonScores) {
+      let score = node.score / 100000000.0;
+      scores[node.public_key] = score;
+    }
+    return scores;
+  }
+
+  async _getNetwork() {
+    const response = await this._grpc.sendCommand('getInfo');
+    return response.chains[0].network;
+  }
```

Comment:
We should set `store.network` in the info action when we poll `getInfo`. No need to do an extra grpc roundtrip here.